### DISCUSSION TITLE
Expanded Digestive System Remove Relivant Mutations

### DISCUSSION THREAD

**Discussion by meelock** [CleverRaven/Cataclysm-DDA#47758] [Ideas]
Realistically the Expanded Digestive System should remove all traits or mutations related to the digestive system, as all of them have to do with the digestive system being replaced. 

The mutations Weak Stomach, Lactose Intolerance, Carnivore, Grain Intolerance, Meat Intolerance, Junk-food Intolerance would re removed on installation of the Expanded Digestive System CBM. Eater of the dead, Intestinal Fortitude and Vomitus might also be replaced, but I don't really know. Also, technically the description doesn't say that the original system is being replaced, rather that it is being augmented, but I don't think this really makes sense.

**Comment by anothersimulacrum** [comment]
This is now the case.
